src/components/Tree/index.js

```diff
@@ -37,24 +57,36 @@ Tree.propTypes = {
         }),
     ),
     /** The action triggered when the user clicks in the tree node expand or collapse button. */
-    onExpandCollapse: PropTypes.func,
+    onNodeExpand: PropTypes.func,
     /** The action triggered when the user clicks in the tree node checkbox. */
-    onSelect: PropTypes.func,
+    onNodeCheck: PropTypes.func,
+    /** The action triggered when the user clicks in the tree node label. */
+    onNodeSelect: PropTypes.func,
+    /** The tree node name. */
+    selectedNode: PropTypes.string,
```

Comment:
@LeandroTorresSicilia this API changes have been discussed?

Reply:
@yvmunayev, Yes it was already discussed, these changes are approved